plugins/inputs/powerdns_recursor/README.md

```diff
@@ -16,8 +16,22 @@ the unix controlsocket.
   # socket_dir = "/var/run/"
   ## Socket permissions for the receive socket.
   # socket_mode = "0666"
+
+  ## The version of the powerdns control protocol to use. You will have to
+  ## change this based on your PowerDNS Recursor version, see below:
+  ## PowerDNS <4.5.0 => Protocol Version 1
+  ## PowerDNS 4.5.0 - 4.5.9 => Protocol Version 2
+  ## PowerDNS >=4.6.0 => Protocol Version 3
```

Comment:
Do you perhaps know if there is a way to detect the version so you don't have to configure it?

Reply:
I can't think of a good way to detect the version, since the protocol itself doesn't carry any version information and these versions numbers are just numbers I came up with. Could do something like try them all progressively and settle on the one that works, but doesn't sound ideal.

Reply:
It currently defaults to version 1 when not set.